README.md

```diff
@@ -1,20 +1,10 @@
 # devfile
 
-## First Generation: chefile
+This repository contains the specs, docs and examples of the devfile.
 
-![chefile](chefile.png)
-[Diagram source (plantuml)](chefile.plantuml)
-
-## Second Generation: chefile.next
-
-Chefile updated with workspace.next model
-
-![chefile.next](chefile.next.png)
-[Diagram source (plantuml)](chefile.next.plantuml)
-
-## Third Generation: devfile
-
-Chefile.next made non che specific
+## Next Generation: devfile
 
 ![devfile](devfile.png)
-[Diagram source (plantuml)](devfile.plantuml)
\ No newline at end of file
+[Diagram source (plantuml)](devfile.plantuml)
```

Comment:
Please add new line here to avoid such displaying
![Screenshot_20190320_122804](https://user-images.githubusercontent.com/5887312/54677488-aa931480-4b0b-11e9-8083-403a6788cff4.png)

Reply:
ok